airflow/providers/openai/provider.yaml

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-openai
+
+name: OpenAI
+
+description: |
+    `OpenAI <https://platform.openai.com/docs/introduction>`__
+
+suspended: false
+
+versions:
+  - 0.0.1
```

Comment:
```suggestion
  - 1.0.0
```

Reply:
@pankajastro We need to have `0.0.1` since the SDK of OpenAI is not production-ready.

Reply:
Airflow provider version is not correlated with SDK version
We have never released 0.0.1 provider.
i don't think there is a problem with doing that (cc @potiuk ?)
but as a general rule don't try to create binding between the two.

Reply:
Starting with 0.0.1 is odd though, “patch” version of an unreleased package :) .

0.1.0 or 1.0.0 is fine for me

If OpenAI releases a breaking change, we can bump up the major version.

Reply:
Changed the version back to 1.0.0.